gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcEventRepository.java

```diff
@@ -397,23 +452,25 @@ private List<Event> queryEvents(String sql, List<Object> args) {
             rowCallbackHandler
         );
         final List<Event> events = rowCallbackHandler.getRows();
-        LOGGER.debug("Events found: {}", events);
+        log.debug("Events found: {}", events);
         return events;
     }
 
     private StringBuilder createSearchQueryBuilder() {
-        final StringBuilder builder = new StringBuilder("select e.*, ep.*, ev.* from " + this.tableName + " e ");
+        final StringBuilder builder = new StringBuilder("select e.*, ep.*, ev.*, evo.* from " + this.tableName + " e ");
         builder.append(" left join ").append(EVENT_PROPERTIES).append(" ep on e.id = ep.event_id ");
         builder.append(" left join ").append(EVENT_ENVIRONMENTS).append(" ev on e.id = ev.event_id ");
+        builder.append(" left join ").append(EVENT_ORGANIZATIONS).append(" evo on e.id = evo.event_id ");
         return builder;
     }
 
-    static void appendCriteria(
+    protected static void appendCriteria(
```

Comment:
```suggestion
    private static void appendCriteria(
```

Reply:
Actually, this is the only thing I was able to reuse in JdbcEventLatestRepository.